docs/reference/slate/change.md

```diff
@@ -53,40 +53,50 @@ function onSomeEvent(event, change) {
 
 `normalize() => Void`
 
-This method normalizes the document with the value's schema. This should run automatically-you should not need to call this method unless you have manually disabled normalization (and you should rarely, if ever, need to manually disable normalization). The vast majority of changes, whether by the user or invoked programmatically, will run `normalize` by default to ensure the document is always in adherence to its schema. `withoutNormalization` also runs `normalize` upon completion.
+This method normalizes the document with the value's schema. This should run automatically-you should not need to call this method unless you have manually disabled normalization (and you should rarely, if ever, need to manually disable normalization). The vast majority of changes, whether by the user or invoked programmatically, will run `normalize` by default to ensure the document is always in adherence to its schema.
 
 > 🤖 If you must use this method, use it sparingly and strategically. Calling this method can be very expensive as it will run normalization on all of the nodes in your document.
 
-### `withoutNormalization`
+### `withoutNormalizing`
 
-`withoutNormalization(customChange: Function) => Change`
+`withoutNormalizing(fn: Function) => Change`
```

Comment:
Sorry if I missed previous discussion on this, but I wonder if it would make sense for this method to pass change as an argument so that they user doesn't have to rely on creating a closure. For example:

```js
import myChangeFunction from './changeHelpers';
change.withoutSaving(myChangeFunction);
```

Same for `withoutSaving()` and `withoutMerging`

Reply:
Hey @ericedem , this is how it works already. Thanks!

Reply:
@CameronAckermanSEL Oh yea good call, thanks!